### PR TITLE
tt 0.0.1 (new formula)

### PR DESCRIPTION
**Pull request by crispgm**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

New formula for <https://github.com/lemnos/tt>

### CODE REVIEW

File: Formula/tt.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Tt < Formula
+  desc "Terminal based typing test"
+  homepage "https://github.com/lemnos/tt"
+  url "https://github.com/lemnos/tt/archive/0.0.1.tar.gz"
```
Comment (SMillerDev) [inline Formula/tt.rb:4]: Is this really a stable release?

File: Formula/tt.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 "007ccfabae069035bf01274b482c84af36b5f0588066da1cd784e62caa461f1e"
+  license "MIT"
+  head "https://github.com/lemnos/tt.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"tt"
```
Comment (SMillerDev) [inline Formula/tt.rb:12]:
```suggestion
    system "go", "build", *std_go_args
```

File: Formula/tt.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  test do
+    assert_match(/nord/, shell_output("#{bin}/tt -list themes"))
```
Comment (SMillerDev) [inline Formula/tt.rb:16]:
```suggestion
    assert_match("nord", shell_output("#{bin}/tt -list themes"))
```

**Comment by fxcoudert** [comment]
Hi @crispgm this is a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by crispgm** [comment]
@fxcoudert thanks for your kindness. I can understand this and I am glad to contribute this to a private tap.

**Pull request closed without merging**